### ISSUE TITLE
Supported devices !!

### ISSUE THREAD

**Issue by iampato** [iampato/sip_native#1]
Even though the specs states the SIP API is available for Android devices from 2.3, it is not. The Android SIP API is not supported on all devices. Interestingly, it is supported on my old 2.3 LG P970 and not my 4.4 Moto G. (By the way, you have to test on real device and not the Android emulator).
Now there are alternatives: 
With the Android SIP API out, what are the alternatives? Well, there are a couple of open source SIP stacks for Android. The popular ones people talk about are:
- JAIN
- MjSIP
- PjSIP
- Doubango

more on this link: https://obem.be/2014/06/04/sip-on-android.html

**Comment by iampato** [comment]
Eventual we will port this plugin to anyone of the above

**Comment by jasonlaw** [comment]
Hi, just want to share with you, do you know [linphone library](https://www.linphone.org/technical-corner/liblinphone)?
Is it something similar to what you are doing?
Thanks!

**Comment by iampato** [comment]
Yes I know Linphone when we were building this we tried Linphone the problem then they had not build the new kotlin SDK 
so someone had to download the lib files and compile them and use them (it never worked)

**Comment by iampato** [comment]
I have gotten time (May the weekend) I shall port the linphone SDK for android/ios/linux/macos/windows

**Comment by iampato** [comment]
we tried pjsip https://github.com/iampato/sip_native/tree/new_pjsip
